Reyes_JohnRuiz/index.html

```diff
@@ -0,0 +1,184 @@
+          <img src="strnger.jpg" alt="Image 6" />
+        </div>
+      </section>
+
+      <section class="my-goals">
+        <h2>My Goals</h2>
+        <ol>
+          <li>To be a Developer</li>
+          <li>To learn some new Skills</li>
+          <li>to travel outside of country </li>
+          <li>to </li>
+          <ol>To be happy</ol>
+        </ol>
+      </section>
+
+      <section>
+        <h2>Add a comment here</h2>
+        <form id="comment-form" class="add-comment">
+          <input
+            type="text"
+            name="name2"
+            id="comment-name"
+            placeholder="Enter your name"
+          />
+          <textarea
+            name="textarea-for-comment"
+            id="comment-text"
+            placeholder="Add your comment here"
+          >
+          </textarea>
+          <button type="button" name="comment" id="submit-comment">Comment</button>
+        </form>
+      </section>
+
+      <section id="comments-container" class="comments-of-team">
+        <h2>Sorted Buttons</h2>
+        <button type="button" id="sort-asc">Sort by Date Ascending</button>
+        <button type="button" id="sort-desc">Sort by Date Descending</button>
+        <h2>New Comments</h2>
+      </section>
+
+      <section id="comments-container" class="comments-of-team">
+        <h2>Hardcoded Comments</h2>
+        <p>Name: Paul Angelo Reyes</p>
+        <p>Comment: You can do it!</p>
+        <p>Name: Shamma Bergado</p>
+        <p>Comment: May you achieve all your goals someday! :></p>
+        <p>Name: Mary Nicole Perillo Norona</p>
+        <p>Comment: Keep on fighting in achieving your dream.</p>
+        <p>Name: Daisy Cangayda Morcoso</p>
+        <p>
+          Comment: I appreciate how your goals are not just about personal
+          success but also about making a positive impact on others and the
+          world around you.
+        </p>
+      </section>
+
+
+      <div class="oc-thoughts">
+        <h2>Thoughts about online class</h2>
+        <p>
+          The reason why is Face-to-face classes are valued over online ones
+          because they provide hands-on learning experiences and real-time
+          interaction, leading to a deeper understanding of the subject matter.
+        </p>
+        <p>
+          Online classes save you from the hassle and expense of commuting in
+          hot weather, providing a comfortable alternative to traveling in the
+          scorching heat. However, without the structure of a traditional
+          classroom, it's easy to get sidetracked and lose focus. It's crucial
+          to remain disciplined and dedicated to your studies to maximize the
+          benefits of this convenient learning option, especially when the cost
+          of transportation and other expenses can add up quickly.
+        </p>
+      </div>
+    </main>
+
+    <div class="Contact">
+      <h2>Contact Me</h2>
+      <form id="contact-form" class="form">
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. See the reference and suggested code for clarifications.
```suggestion
      <form id="contact_form" class="form">
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5